apps/dashboard/src/components/workflow-row.tsx

```diff
@@ -160,47 +180,43 @@ export const WorkflowRow = ({ workflow }: WorkflowRowProps) => {
   };
 
   return (
-    <TableRow key={workflow._id} className="relative">
+    <TableRow key={workflow._id} className="group relative isolate">
```

Comment:
What if we just do:

```
    <TableRow key={workflow._id} className="hover:bg-accent/50 relative hover:cursor-pointer" onClick={handleRowClick}>
```

On the main row, and add to the onHoverCopy and actions cell:
```
          onClick={(e) => e.stopPropagation()}
```

Might simplify this a bit, wdyt?

Reply:
The implementation feels like an overkill, but if I am not mistaken, it's the only way to maintain valid Table/tr/td HTML while treating each row as a link, making right-clicking Open in a new Tab work.

@desiprisg Please confirm.

Reply:
Yes that's the thinking behind this. Also checked with some other implementations like stripe's and they seem to follow the same approach.